contracts/test/stateverifier.test.ts

```diff
@@ -134,13 +152,20 @@ describe("Verify", function () {
     expect(resultFromStarknetVerifier.toString()).to.be.eq(result[0]);
   });
 
+  it(" test for a poseidon array", async function () {
+    const result = await l1resolverStub.poseidonHashMany(["28355430774503553497671514844211693180464", "0x34894aedf9548524f9e5bb189472d25abe3c38befd577c90886a7c519e5eee4", "0x70c5acad61a421be9c2945b921e263f2699c668f20d31c90660e91b32ea99de"]);
+    console.log("Result: poseidon array ", result);
+    expect(result).to.eq("0x7d608b218ca14fd5174b0fc40a9ea0c34bf7f855533323f348c47c9aa89fe90");
+  });
+
+
   it("it should verify the sampleproof1.json", async function () {
     var jsonFile = "./sampleProof1.json";
     const contractAddress =
-      "0x6fbd460228d843b7fbef670ff15607bf72e19fa94de21e29811ada167b4ca39";
+      "0x7412b9155cdb517c5d24e1c80f4af96f31f221151aab9a9a1b67f380a349ea3";
```

Comment:
`L2_RESOLVER_ADDRESS`

Reply:
no, this can be any contract, doesn't have to be a L2_RESOLVER_ADDRESS. these are just verifying proofs for a storage value for any contract.